src/App.js

```diff
@@ -258,14 +258,14 @@ class App extends Component {
         <Footer snippets={{ ...snippets.footer, langCode: snippets.langCode }} />
 
         { (devProps.dev === true) ? (
-            <DevHud
-              devProps = { devProps }
-              funcs    = { devFuncs }
-              data     = {{ default: clients.default }}
-              state    = { this.state } />
-          ) : (
-            null
-          )
+          <DevHud
+            devProps = { devProps }
+            funcs    = { devFuncs }
+            data     = {{ default: clients.default }}
+            state    = { this.state } />
+        ) : (
+          null
+        )
```

Comment:
`var/let/const` stuff looking good so far! This doesn't happen when I do `npm run lint`, though. Where is this coming from?

Reply:
Or maybe I'm just wrong... I think my editor's stopped showing me lint errors.

Reply:
Regardless, that's not what I wanted that to look like. That said, it's also not this PR's problem.